CineElorrietaMVC/src/main/java/Controlador/ControladorPanelPedidos.java

```diff
@@ -53,8 +69,21 @@ public void setContador(String cont) {
 
 	public String accionadoBottonEliminarTotal() {
 		return modelo.pedidos.eliminarTotal();
-	}	
-	public void accionadoBottonEliminarPanelPedidos(int i) {
-		modelo.pedidos.setArrSelec(i);
+	}
+
+	
+
+	public void insertPedido(String entrega) throws SQLException {
+		modelo.ScriptsBDD.registrarPedido(entrega);
+	}
+
+	public void borrarTotalTickets() {
+		modelo.pedidos.borrarTotal();
+	}
+	
+public String getNombreLocal() throws SQLException {
+	String usuArr = modelo.BDD.getUsuString();
```

Comment:
¿Aquí no sería mejor que traigáis del modelo el nombre del usuario solamente y así no tener que separar nada?
Revisadlo también en tickets/facturas

Reply:
Hecho.